src/hooks/useRecordingContext.test.ts

```diff
@@ -71,20 +69,20 @@ describe('useRecordingContext', () => {
 
   it('should toggle the recording status when toggleRecording is called', async () => {
     const renderHookResponse = renderHook(() =>
-      useRecordingContext(ipc, '', 0, setResult, setSteps)
+      useRecordingContext(electronApi, '', 0, setResult, setSteps)
     );
     const recordingContext = renderHookResponse.result.current;
 
     await act(async () => recordingContext.toggleRecording());
 
     renderHookResponse.rerender();
 
-    expect(callMainMock).toHaveBeenCalledWith('record-journey', { url: '' });
+    expect(recordJourney).toHaveBeenCalledWith('');
```

Comment:
It's outside the scope of this patch but could you do me a favor and declare a `url` at the start of this test, supply it to the hook, and verify that `recordJourney` was called with that URL? I think now that the args the mock receives are just a string it has made the test's intentions less obvious, since the assertion has become just an empty string.

```typescript
const url = 'https://elastic.co';

// ...

useRecordingContext(electronApi, url, setResult, setSteps);

// ...

expect(recordJourney).toHaveBeenCalledWith(url);
```

Reply:
Yup I've updated as you suggested and the test seems to work!

Reply:
Thank you, I think it makes the test more readable.